client/osal/__init__.py

```diff
@@ -0,0 +1,47 @@
+"""Operating System Abstraction Layer"""
```

Comment:
Please install pre-commit hooks as described in README.md:
> pip install pre-commit
> pre-commit install
and clean up the code locally

Reply:
Hooks installed 👍